pyuvdata/parameter.py

```diff
@@ -734,9 +734,12 @@ def __init__(
         value=None,
         spoof_val=None,
         description="",
+        frame="itrs",
```

Comment:
This needs to be added to both the `Parameters` and `Attributes` section of the docstring.

Reply:
done!